modules/combined/test/tests/sliding_block/in_and_out/constraint/sm/tests

```diff
@@ -0,0 +1,108 @@
+[Tests]
+  [frictional_lm_sm]
+    type = 'Exodiff'
+    input = 'frictional_lm_sm.i'
+    exodiff = 'frictional_lm_sm_exodus.e'
+    heavy = true
+    abs_zero = 1e-8
+    design = 'Constraints/index.md'
+    issues = '#13080'
+    requirement = 'We shall be able to run our canonical frictional sliding block problem with mortar Lagrange Multiplier contact'
```

Comment:
Put something about using solid_mechanics at the end of these requirement lines to differentiate them from the tm versions.